shared-libs/settings/test/index.spec.js

```diff
@@ -0,0 +1,137 @@
+const sinon = require('sinon');
+const request = require('request-promise-native');
+const chai = require('chai');
+const { assert, expect } = chai;
+
+const lib = require('../src/');
+
+describe('Settings Shared Library', () => {
+  const environment = {};
+
+  beforeEach(() => {
+    sinon
+      .stub(process, 'env')
+      .value(environment);
+  });
+
+  afterEach(() => sinon.restore());
+
+  describe('getCredentials', () => {
+
+    it('should throw error if no server url is set', () => {
+      environment.COUCH_URL = '';
+      environment.COUCH_NODE_NAME = '';
+
+      return lib
+        .getCredentials()
+        .then(() => assert.fail('exception expected'))
```

Comment:
If I'm not mistaken. `expect` also has a `fail` method, so we could save an import if this is the only thing you're using `assert` for.

Reply:
Done